_pages/DecodeDshot2bit.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title: Decode_Dshot_2bit MACRO
+permalink: /decode-dshot-2bit/
+exclude: true
+---
+
+[![Decode_Dshot_2bit]({{'/puml/svgs/DetailedDesign-DecodeDshot2bit.svg' | relative_url}})]
```

Comment:
This is how the link should look like - sorry, I should have been more clear the first time around.
```
[![Decode_Dshot_2bit]({{'/puml/svgs/DetailedDesign-DecodeDshot2bit.svg' | relative_url}})]({{'/puml/svgs/DetailedDesign-DecodeDshot2bit.svg' | relative_url}})
```

Reply:
fixed